hros5:
# Position controllers ---------------------------------------

 # HEAD

  HeadYaw_position_controller:
    type: position_controllers/JointPositionController
    joint: HeadYaw

  HeadPitch_position_controller:
    type: position_controllers/JointPositionController
    joint: HeadPitch


  # ARMS

  LShoulderPitch_position_controller:
    type: position_controllers/JointPositionController
    joint: LShoulderPitch

  LShoulderRoll_position_controller:
    type: position_controllers/JointPositionController
    joint: LShoulderRoll

  LElbowPitch_position_controller:
    type: position_controllers/JointPositionController
    joint: LElbowPitch

  RShoulderPitch_position_controller:
    type: position_controllers/JointPositionController
    joint: RShoulderPitch

  RShoulderRoll_position_controller:
    type: position_controllers/JointPositionController
    joint: RShoulderRoll

  RElbowPitch_position_controller:
    type: position_controllers/JointPositionController
    joint: RElbowPitch


  # LEGS

  LHipYaw_position_controller:
    type: position_controllers/JointPositionController
    joint: LHipYaw

  LHipRoll_position_controller:
    type: position_controllers/JointPositionController
    joint: LHipRoll

  LHipPitch_position_controller:
    type: position_controllers/JointPositionController
    joint: LHipPitch

  LKneePitch_position_controller:
    type: position_controllers/JointPositionController
    joint: LKneePitch

  LAnklePitch_position_controller:
    type: position_controllers/JointPositionController
    joint: LAnklePitch

  LAnkleRoll_position_controller:
    type: position_controllers/JointPositionController
    joint: LAnkleRoll

  RHipYaw_position_controller:
    type: position_controllers/JointPositionController
    joint: RHipYaw

  RHipRoll_position_controller:
    type: position_controllers/JointPositionController
    joint: RHipRoll

  RHipPitch_position_controller:
    type: position_controllers/JointPositionController
    joint: RHipPitch

  RKneePitch_position_controller:
    type: position_controllers/JointPositionController
    joint: RKneePitch

  RAnklePitch_position_controller:
    type: position_controllers/JointPositionController
    joint: RAnklePitch

  RAnkleRoll_position_controller:
    type: position_controllers/JointPositionController
    joint: RAnkleRoll
